modules/abstract-utxo/src/parseOutput.ts

```diff
@@ -228,6 +229,21 @@ export async function parseOutput({
      * details are fetched on the wallet, a local address validation is run, whose errors however are generated
      * client-side and can therefore be analyzed with more granularity and type checking.
      */
+
+    /**
+     * In order to minimize API requests, we assume that explicit recipients are always external when the
```

Comment:
Is this something that would also be helpful to add in developers.bitgo.com?

Reply:
Maybe, but it might only add to confusion.